recipes/xorg/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class ConanXOrg(ConanFile):
+    name = "xorg"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    homepage = "https://www.x.org/wiki/"
+    description = "The X.Org project provides an open source implementation of the X Window System."
+    settings = {"os": "Linux"}
+
+    def package_id(self):
+        self.info.header_only()
+
+    def _libs_pkgconfig(self, name):
+        pkg_config = tools.PkgConfig(name)
+        if not pkg_config:
+            raise ConanException("%s development files aren't available, give up" % name)
+        libs = [lib[2:] for lib in pkg_config.libs_only_l]
+        return libs
+
+    def system_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
```

Comment:
```suggestion
        if tools.os_info.is_linux:
```

line 11 only allows Linux

Reply:
I'd like to keep it, if we ever add support for FreeBSD/NetBSD/OpenBSD, which will use the same logic

Reply:
(also macOS which may have X11 installed via Xquartz)
and in both cases, the same package (which is literally a package containing zero files) should be available for Linux/*BSD/macOS/whatever